content/creator/sdk7/getting-started/using-the-cli.md

```diff
@@ -13,7 +13,7 @@ weight: 15
 To build scenes for Decentraland you an either use 
 
 - The Decentraland Editor
-- The Command Line Interface (CLI)
+- The CLI: @dcl/sdk-commands global library
```

Comment:
```suggestion
- The CLI: @dcl/sdk-commands global library
```

I would suggest anything related to a global package or something like that. Now all the commands work with `npm run {command}`, and lives inside the scene (except for init). So you don't have to install anything extra